test/helpers/kubectl.go

```
@@ -572,6 +581,47 @@ func (kub *Kubectl) Delete(filePath string) *CmdRes {
 		fmt.Sprintf("%s delete -f  %s", KubectlCmd, filePath))
 }
 
+// HeapsterDeploy deploys Heapster monitoring pods in the given Kubernetes cluster
+func (kub *Kubectl) HeapsterDeploy() error {
+
+	for _, manifest := range heapsterYamls {
+		res := kub.Apply(manifest)
+		if !res.WasSuccessful() {
+			return fmt.Errorf("cannot install Heapster %s: %s", manifest, res.CombineOutput())
```

Since we fail hard on not installing heapster, perhaps we should copy the yamls into our repo? It seems more susceptible to network blips... and random reorganisations of their repo. Since the url includes a micro version, I also imagine the file will never change.